index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
```

Comment:
I suggest that you should use linters so as to make your code clean and easily read by other programmers.